docs/in-depth/decky-loader.md

```diff
@@ -0,0 +1,16 @@
+Decky Loader
+============
+
+> [!INFO]
+> This page is a stub.
+
+To enable *Decky Loader* in your configuration:
+
+```nix
+{
+  jovian.decky-loader = {
+    enable = true;
+    user = "your-username-here";
```

Comment:
Should probably not tell people to do that because upstream still prefers to run as root.